scripts/install

```diff
@@ -87,6 +87,14 @@ ynh_script_progression --message="Configuring system user..." --weight=2
 # Create a system user
 ynh_system_user_create --username=$app
 
+if grep -q render /etc/group
```

Comment:
This would catch string containing `render`, like `renderz`, `zrender`, etc. And even any user containting this string. You should use a regex I think, something like `^render:`